src/sys/aio.rs

```diff
@@ -1184,9 +1187,8 @@ impl<'a> LioCb<'a> {
     ///
     /// [`lio_listio`](http://pubs.opengroup.org/onlinepubs/9699919799/functions/lio_listio.html)
     ///
-    /// [`AioCb::aio_return`] (struct.AioCb.html#method.aio_return)
-    /// [`AioCb::error`] (struct.AioCb.html#method.error)
-    /// [`LioCb::aio_return`] #method.aio_return
+    /// [`lio_listio`]: http://pubs.opengroup.org/onlinepubs/9699919799/functions/lio_listio.html
```

Comment:
Why doesn't this use the parens format for external URL like on line 1188? Is there a reason to prefer one syntax over the other and/or is this actually incorrect?

Reply:
In this rare case, I actually got it right.  Line 1188 is an inline link.  Lines 1190-1191 are footnotes.